crypto/batchverifier_test.go

```diff
@@ -124,3 +125,66 @@ func TestEmpty(t *testing.T) {
 	bv := MakeBatchVerifier()
 	require.NoError(t, bv.Verify())
 }
+
```

Comment:
we can add the `TestEmpty` verification that the `failed` array has zero elements.